app/src/main/java/org/stepic/droid/base/CoursesDatabaseFragmentBase.java

```diff
@@ -15,7 +15,6 @@
 import org.stepic.droid.analytic.Analytic;
 import org.stepic.droid.core.dropping.contract.DroppingListener;
 import org.stepic.droid.core.presenters.PersistentCourseListPresenter;
-import org.stepic.droid.model.Course;
```

Comment:
Why was this import removed? `org.stepic.droid.model.Course course` constructions look strange.